lib/Crust/Request.pm6

```diff
@@ -84,7 +84,7 @@ method header(Str $name) {
 method content() {
     # TODO: we should support buffering in Crust layer
     my $input = $!env<p6sgi.input>;
-    $input.seek(0,0); # rewind
+    $input.seek(0); # rewind
```

Comment:
I already sent PR https://github.com/moznion/p6-IO-Blob/pull/6, however this is not enough(Default value is not specified). I'll fix later.